src/app/view/components/_common/iframe/iframe-porlet-legacy/iframe-porlet-legacy.component.ts

```diff
@@ -44,7 +47,20 @@ export class IframePortletLegacyComponent implements OnInit {
                 this.reloadIframePortlet();
             }
         });
-        this.setIframeSrc();
+
+        this.route.data
+            .pipe(pluck('canAccessPortlet'), takeUntil(this.destroy$))
+            .subscribe((canAccessPortlet: boolean) => {
+                if (canAccessPortlet) {
+                    this.setIframeSrc();
+                }
+                this.canAccessPortlet = canAccessPortlet;
```

Comment:
This is like a doble flag here, I mean if you already doing `setIframeSrc` use whatever is get set there to show the iframe or the license.

Reply:
when tried to use `url` as a param to it occurs undesirable behaviour that makes the Not Licensed component not render when switching between different enterprise portlets.